src/it/scala/io/iohk/ethereum/ledger/BlockImporterItSpec.scala

```diff
@@ -62,6 +65,18 @@ class BlockImporterItSpec
       ethCompatibleStorage = true
     )
 
+  override protected lazy val successValidators: Validators = new Mocks.MockValidatorsAlwaysSucceed {
+    override val ommersValidator: OmmersValidator = (
+        parentHash: ByteString,
+        blockNumber: BigInt,
+        ommers: Seq[BlockHeader],
+        getBlockHeaderByHash: GetBlockHeaderByHash,
+        getNBlocksBack: GetNBlocksBack
+    ) =>
+      new StdOmmersValidator(blockchainConfig, blockHeaderValidator)
+        .validate(parentHash, blockNumber, ommers, getBlockHeaderByHash, getNBlocksBack)
+  }
```

Comment:
I know it's a test but what's the gain of `val` instead of `def`?

Reply:
val is the only option because it's overriding a val from MockValidatorsAlwaysSucceed

Reply:
gotcha, thanks